main.py

```diff
@@ -57,7 +57,7 @@ def respond(voice_data):
         webbrowser.get().open(url)
         alina_speak(f"Here is what I found for {search}")
     if "what is your name" in voice_data:
-        alina_speak("My name is Alina.")
+        alina_speak("My name is Tejax.")
```

Comment:
Instead of adding a function to change the assistant's name to what the user desires, you have just replaced the previous name with your name